examples/bri-3/src/bri/communication/messagingClients/messagingClient.interface.ts

```diff
@@ -0,0 +1,7 @@
+export interface IMessagingClient {
```

Comment:
similar comment as above, are we abstracting something for no reason atm? are we sure that other messaging clients can be reduced to these 2 simple methods?

Reply:
This interface marks the boundary between the BPI domain language and the outside world so it makes sense to keep it here even if we remain with nats client only. 
 
It should be possible to reduce  every pub\sub client to these. In case it is not, we will redesign the interface.